test/from-test.js

```diff
@@ -0,0 +1,90 @@
+var expect = require('chai').expect;
+
+var zstreams = require('../lib');
+
+describe('zstreams.fromArray', function() {
+	it('should create an object stream from an array', function(done) {
+		var input = ['a', 'b', 'c', 'd'];
+		var arrayStream = zstreams.fromArray(input)
+		arrayStream.toArray(function(error, array) {
+			expect(error).to.not.exist;
+			expect(arrayStream.isReadableObjectMode()).to.equal(true);
+			expect(array).to.be.instanceof(Array);
+			expect(array).to.have.length(input.length);
+
+			input.forEach(function(entry, i) {
+				expect(array[i]).to.equal(entry);
+			});
+
+			done();
+		});
+	});
+});
+
+describe('zstreams.fromFile', function() {
+	it('should create a data stream from a file', function(done) {
+		var fileStream = zstreams.fromFile(__dirname + '/resources/abcd.json');
+		fileStream.toArray(function(error, array) {
```

Comment:
To examine the output of whatever is coming through.  
A couple of these tests should probably be rewritten using `intoString`. It's just `toArray` is the easiest way to flush the output to a usable format.